src/core.jl

```diff
@@ -1,16 +1,18 @@
 export DSSCore
 
 "Module with the low-level API for OpenDSSDirect"
-module DSSCore 
+module DSSCore
 
 if is_windows()
     if Int == Int64
         const dsslib = "$(dirname(@__FILE__))\\..\\deps\\win64\\OpenDSSDirect.DLL"
     else
         const dsslib = "$(dirname(@__FILE__))\\..\\deps\\win32\\OpenDSSDirect.DLL"
     end
-else
-    const dsslib = "$(dirname(@__FILE__))/../deps/libopendssdirect.so"
+elseif is_linux()
```

Comment:
Right but Julia supports non-glibc Linux distributions, as well as 32 bit x86, 32 and 64 bit ARM, and 64 bit Power. `is_linux()` will be true for all of them.